egs/ami/s5/local/run_cleanup_segmentation.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+# This script demonstrates how to re-segment training data selecting only the 
+# "good" audio that matches the transcripts.
+# The basic idea is to decode with an existing in-domain acoustic model, and a
+# biased language model built from the reference, and then work out the
+# segmentation from a ctm like file.
+
+# For nnet3 and chain results after cleanup, see the scripts in 
+# local/nnet3/run_tdnn.sh and local/chain/run_tdnn.sh 
+
+# GMM Results:
+
```

Comment:
I don't understand the significance of the results below.
There are 4 repeated sets of lines with the same directory name, and no indication of what the different results mean.
BTW, it's not enough to make the directory names different, there needs to be something that makes this understandable to anyone except you.